doc/source/serve/deploy-many-models/model-multiplexing.md

```diff
@@ -72,3 +71,18 @@ INFO 2023-05-24 01:19:15,988 default_Model default_Model#rimNjA WzjTbJvbPN / def
 INFO 2023-05-24 01:19:15,988 default_Model default_Model#rimNjA WzjTbJvbPN / default multiplex.py:131 - Loading model '4'.
 INFO 2023-05-24 01:19:16,993 default_Model default_Model#rimNjA WzjTbJvbPN / default replica.py:542 - __CALL__ OK 1005.7ms
 ```
+
+
+You can also send a request to a specific model by using handle {mod}`options <ray.serve.handle.RayServeHandle>` API.
+```{literalinclude} ../doc_code/multiplexed.py
+:language: python
+:start-after: __serve_handle_send_example_begin__
+:end-before: __serve_handle_send_example_end__
+```
+
+By using ray serve handle, you can also send multiplexed request within a deployment for model composition.
```

Comment:
Do you have to set this when using model composition or is it an override? eg. if you don't set it through the handle options, is it just able to use the value from the header of the original request? This should be made more clear

Reply:
Added a sentence to mention you should always set `multiplexed_model_id` when using ray serve handle.